src/model/CustomerDealer.java

```diff
@@ -0,0 +1,109 @@
+package model;
+
+import constant.CustomerDealerEnum;
+import java.sql.Date;
+import java.time.LocalDate;
+
+public class CustomerDealer {
+
+    private int customerDealerId;
+    private String customerDealerName;
+    private CustomerDealerEnum customerDealerEnum;
+    private String email;
+    private String phoneNumber;
+    private String address;
+    private Date createDate;
+    private Date modifiedDate;
+    private String createdBy;
+    private String updatedBy;
+    private int active;
```

Comment:
make active as boolean.

Reply:
aggred and also connect with @TouqeerAli  for auditable class